Source/Scene/PointCloud3DTileContent.js

```diff
@@ -412,10 +427,22 @@ define([
             this.batchTable = new Cesium3DTileBatchTable(this, batchLength, batchTableJson, batchTableBinary);
         }
 
-        // If points are not batched and there there are per-point properties, use these properties for styling purposes
+        // If points are not batched and there are per-point properties, use these properties for styling purposes
         var styleableProperties;
         if (!defined(batchIds) && defined(batchTableBinary)) {
             styleableProperties = Cesium3DTileBatchTable.getBinaryProperties(pointsLength, batchTableJson, batchTableBinary);
+
+            // WebGL does not support UNSIGNED_INT, INT, or DOUBLE vertex attributes. Convert these to FLOAT.
```

Comment:
Does `float` start to skip whole numbers in the max `UNSIGNED_INT`/`INT` range?  If so, perhaps we should `console.log` warn.  Otherwise, we should just warn for doubles.

For the spec, I think we'll need to allow conversion to float so that styling is implementable with WebGL 1.0 shaders.